examples/tutorial.py

```diff
@@ -0,0 +1,434 @@
+            self.axis_point,
+            self.radius*sm.cos(self.insertion_angle)*self.child_axis.cross(self.axis)
+            + self.radius*sm.sin(self.insertion_angle)*self.child_axis),
+
+        parent_force_direction_vector = self.origin.pos_from(parent_tangency_point)
+        child_force_direction_vector = self.insertion.pos_from(child_tangency_point)
+        force_on_parent = force_magnitude*parent_force_direction_vector.normalize()
+        force_on_child = force_magnitude*child_force_direction_vector.normalize()
+        loads = [
+            me.Force(self.origin, force_on_parent),
+            me.Force(self.axis_point, -(force_on_parent + force_on_child)),
+            me.Force(self.insertion, force_on_child),
+        ]
+        return loads
+
+tricep_pathway = ExtensorPathway(Cm, Dm, P3, B.y, -C.z, D.z, r, q4)
+tricep_activation = bm.FirstOrderActivationDeGroote2016.with_default_constants('tricep')
+tricep = bm.MusculotendonDeGroote2016('tricep', tricep_pathway,
+                                      activation_dynamics=tricep_activation)
+
+tricep.to_loads()
+
+loads = (
+    bicep.to_loads() +
+    tricep.to_loads() +
+    [lever_resistance, gravC, gravD]
+)
+
+kane = me.KanesMethod(
+    N,
+    (q1,),
+    (u1,),
+    kd_eqs=(
+        u1 - q1.diff(),
+        u2 - q2.diff(),
+        u3 - q3.diff(),
+        u4 - q4.diff(),
+    ),
+    q_dependent=(q2, q3, q4),
+    configuration_constraints=holonomic,
+    velocity_constraints=holonomic.diff(me.dynamicsymbols._t),
+    u_dependent=(u2, u3, u4),
+)
+Fr, Frs = kane.kanes_equations((lever, u_arm, l_arm), loads)
+
+kane.mass_matrix
+
+kane.forcing
+
+
+me.find_dynamicsymbols(kane.forcing)
+
+
+kane.forcing.free_symbols
+
+bicep.activation_dynamics.state_equations
+
+tricep.activation_dynamics.state_equations
+
+# TODO : these are dictionaries and we don't have easy access to the key, so
+# not really a clean way to extract the equation it doesn't mesh well with the
+# rest of mechanics API. If an acutator has dynamics, and thus state, then the
+# general form of the equations should be something like:
+# M(x, r, t) x' = F(x, r, t)
+# x being the state vector (sympy column matrix)
+# r being the input vector (sympy column matrix)
+# To mesh with .mechanics the Actuator should likely have these
+# methods/attributes:
+# Actuator.x or Acuator.state_vars returns an ordered column matrix of
+# functions of time that represent the state variables
+# Acutator.r or Acutors.input_vars returns and ordered columan matrix of
+# functions of time that represnet the input variabls (I use r here because we
+# use u for generalized
+# speeds in mechanics
+# Acutator.p Acuator.constants returns a columan matrix of orderd non-time
+# varying symbols present in M and F.
+# Actuator.M returns a matrix matching the order of x
+# Actuator.F returns a matrix matching the order of x
+# Acuator.rhs() returns a matrix that is solve(self.M, self.F) matching the
+# order of x
```

Comment:
@brocksam  Let me know what you think of this. I found accessing things here a bit awkward and not so aligned with existing API.